01-front-end-basics/html-forms/25/exercises.md

```diff
@@ -0,0 +1,49 @@
+**Forms & Queries** -- Exercises
+
+Please clone and use this [blank template](https://github.com/sf-wdi-25/blank_template) as your starting point.
```

Comment:
Here's the most recent template: https://github.com/SF-WDI-LABS/blank_template

Eventually it might make sense to move the exercises into a lab repo based on the template, so they have a stronger starting point.